### ISSUE TITLE
SVG backend ignores interpolation='nearest' option in imshow()

### ISSUE THREAD

**Issue by tkanmae** [matplotlib/matplotlib#4136]
I have raster images and plotted them using `imshow()` with `interpolation='nearest`. But the resultant images look smeared as if they were plotted with `interpolation='bilinear'`. Note that I set `svg.image_noscale` to `True`.  I briefly checked with the PDF and PNG backends, but these backends do not have this issue.

I'm using the version 1.4.3 with Python 3.4.2 on Mac OS X.

**Comment by mdboom** [comment]
The interpolation used in the SVG viewer is usually hardcoded -- it can't be specified in the file (it's usuall bilinear or bicubic depending).

So if you want the effect of `interpolation='nearest'` you'll have to let matplotlib scale up the image itself before putting it in the file.  `svg.image_noscale = True` means that matplotlib will never scale the image, but just put the image verbatim in the file, but then you are at the mercy of whatever the SVG viewer wants to do to scale it.

So in short: try setting `svg.image_noscale` back to `False`.  If that doesn't work, feel free to reopen this issue, with an example script to reproduce and an SVG file if possible.

**Comment by tkanmae** [comment]
Thanks for the clarification. Setting `svg.image_noscale` to `False` worked.

**Comment by tkanmae** [comment]
If I understand your explanation correctly, I think the comment for `svg.image_noscale` would be more clear if it states "request SVG viewers not to scale raster data embedded in a file" rather than "suppress scaling of raster data embedded in SVG".
